test/access-control.integration.js

```diff
@@ -163,7 +169,7 @@ describe('access control - integration', function() {
     }
   });
 
-  describe('/accounts', function() {
+  describe('/accounts with replaceOnPUT true', function() {
```

Comment:
Oh, I should have read the full patch before commenting. So you actually did what I proposed in `/accounts with replaceOnPUT false`. Can we please simplify that other describe block and keep there only tests relevant to `replaceOnPut`?